components/timezone/src/types.rs

```diff
@@ -93,32 +93,24 @@ impl GmtOffset {
             _ => return Err(TimeZoneError::InvalidOffset),
         };
 
-        let seconds = match chars {
+        let hours = match chars {
+            &[h1, h2] => try_get_time_component([h1, h2]),
```

Comment:
issue: won't this fail to match `[h1, h2, m1, m2]`?

Reply:
whoops